community/knot-resolver/APKBUILD

```diff
@@ -16,17 +16,18 @@ depends="dns-root-hints dnssec-root lua5.1-sec lua5.1-socket"
 _depends_dnstap_dev="fstrm-dev protobuf-dev protobuf-c-dev"
 _depends_http="$pkgname lua5.1-http"
 depends_dev="knot-dev>=2.8.0 libedit-dev libuv-dev>=1.7 lmdb-dev luajit-dev>=2.0 $_depends_dnstap_dev"
+depends_static="$pkgname-dev"
 makedepends="$depends_dev bash cmake gnutls-dev luacheck meson>=0.46 ninja pkgconf py3-flake8"
 checkdepends="cmocka-dev"
 install="$pkgname.pre-install"
-subpackages="$pkgname-mod-http:http:noarch $pkgname-mod-dnstap:dnstap $pkgname-dev $pkgname-doc $pkgname-openrc"
+subpackages="$pkgname-mod-http:http:noarch $pkgname-mod-dnstap:dnstap $pkgname-libs-static $pkgname-dev $pkgname-doc $pkgname-openrc"
 source="
-	https://secure.nic.cz/files/${_pkgname}/${_pkgname}-$pkgver.tar.xz
+	https://secure.nic.cz/files/${_upstream}/${_upstream}-$pkgver.tar.xz
```

Comment:
@Ikke asked to use the written out name as noted [here](https://github.com/maxice8/atools/blob/master/alint.5.scd#L316)